### PR TITLE
update keadm

### PR DESCRIPTION
**Pull request by zhu733756**
Signed-off-by: zhu733756 <zhu733756@kubesphere.io>

* **Please check if the PR fulfills these requirements**
- [x] The commit message follows our guidelines
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

Update keadm introduction to implementing keadm beta features.

* **What is the current behavior?** (You can also link to an open issue here)

* **What is the new behavior (if this is a feature change)?**

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)
None, the former cmds all supported.

### CODE REVIEW

File: content/en/docs/setup/keadm.md [hunk 1]
```diff
@@ -22,6 +22,8 @@ Please refer [kubernetes-compatibility](https://github.com/kubeedge/kubeedge#kub
 
 By default ports `10000` and `10002` in your cloudcore needs to be accessible for your edge nodes.
 
+### keadm init 
+
 `keadm init` will install cloudcore, generate the certs and install the CRDs. It also provides a flag by which a specific version can be set.
```
Comment (fisherxu) [inline content/en/docs/setup/keadm.md:27]:
```suggestion
`keadm init` will install cloudcore, generate the certs and install the CRDs. It also provides a flag by which a specific version can be set.

**Note:** `keadm init` deploy cloudcore in binary process as system service, if you want to deploy cloudcore as container, please ref `keadm beta init` below.
```

Let's add some notes for users. :)

File: content/en/docs/setup/keadm.md [hunk 2]
```diff
@@ -42,6 +44,36 @@ Kubernetes version verification passed, KubeEdge installation will start...
...
 ```
 
+### keadm beta init
+
+We now provide a solution for integrating Cloudcore Helm Chart with `keadm beta init`. 
+
+Example:
+
+```shell
+# keadm beta init --advertise-address="THE-EXPOSED-IP" [--set cloudcore-tag=v1.9.0] --profile version=v1.9.0 --kube-config=/root/.kube/config
```
Comment (fisherxu) [inline content/en/docs/setup/keadm.md:54]: What's meaning of `[]` for `--set`? And if we want to specify the kubeedge version, we need to modify both the `[--set cloudcore-tag=v1.9.0]` and `--profile version=v1.9.0`?

File: content/en/docs/setup/keadm.md [hunk 3]
```diff
@@ -42,6 +44,36 @@ Kubernetes version verification passed, KubeEdge installation will start...
...
+If you are familiar with the helm chart installation, please refer to [KubeEdge Helm Charts](https://github.com/kubeedge/kubeedge/tree/master/build/helm/charts).
+
+### keadm beta manifest generate
+
+You can also get the manifests with `keadm beta manifest generate`.
+
+Example:
+
+```shell
+# keadm beta manifest generate --advertise-address="THE-EXPOSED-IP" --kube-config=/root/.kube/config [--skip-crds]
```
Comment (fisherxu) [inline content/en/docs/setup/keadm.md:74]: Where will the manifests generated to?

File: content/en/docs/setup/keadm.md (hunk 2 above)
Comment (zhu733756) [reply content/en/docs/setup/keadm.md:54]: Set flags would override the value from values.yaml with profile. I would remove these confusing values.

File: content/en/docs/setup/keadm.md (hunk 3 above)
Comment (zhu733756) [reply content/en/docs/setup/keadm.md:74]: Print the outputs on the stdout.

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**